test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -52,40 +50,39 @@ window.dataLayer = [];
 describe('<AddressSection>', () => {
   // we expect a render with default props to show the AddressContent component
   it('should display an address if one is provided in props', () => {
-    const tree = SkinDeep.shallowRender(<AddressSection {...defaultProps}/>);
-    const addressContent = tree.dive(['AddressContent']);
-    const contentHeader = addressContent.subTree('p').text();
+    const tree = shallow(<AddressSection {...defaultProps}/>);
+    const addressContent = tree.find('AddressContent');
+    const contentHeader = addressContent.dive().find('p').text();
 
     expect(contentHeader).to.contain('Downloaded documents will list your address as:');
   });
 
   it('should display a loading spinner if address save in progress', () => {
     const newProps = { ...defaultProps, savePending: true };
-    const tree = SkinDeep.shallowRender(<AddressSection {...newProps}/>);
-    const spinner = tree.dive(['AddressContent', 'AddressBlock', 'LoadingIndicator']);
+    const tree = shallow(<AddressSection {...newProps}/>);
+    const spinner = tree.find('AddressContent').dive()
+      .find('AddressBlock').dive()
+      .find('LoadingIndicator')
+      .dive();
     expect(spinner.text()).to.contain('Updating your address...');
   });
 
   it('should format 1 address line', () => {
-    const tree = SkinDeep.shallowRender(<AddressSection {...defaultProps}/>);
-    const addressBlock = tree.dive(['AddressContent', 'AddressBlock']);
-    const addressBlockText = addressBlock.subTree('.address-block', 'div.letters-address').text();
-    // NOTE: have to pass in uncapitalized addresses to assert against because shallowRender apparently strips caps...
+    const tree = shallow(<AddressSection {...defaultProps}/>);
+    const addressBlock = tree.find('AddressContent').dive()
+      .find('AddressBlock').dive();
```

Comment:
Do you need the additional `.dive()` at the end here? Tried pulling the branch and running it locally myself to test it, but getting some console errors.

Reply:
Yeah, I think because without it, the `.find()` gets a `ShallowRenderWrapper` (or something) and not the "actual" component itself.

Super technical description, right? At any rate, I tried to use as few `dive()`s as possible, and it didn't work without that one. 😞